src/components/include/policy/policy_regular/policy/policy_manager.h

```diff
@@ -502,6 +502,69 @@ class PolicyManager : public usage_statistics::StatisticsManager {
    */
   virtual const VehicleInfo GetVehicleInfo() const = 0;
 
+  /**
+   * @brief Get a list of enabled cloud applications
+   * @param enabled_apps List filled with the policy app id of each enabled
+   * cloud application
+   */
+  virtual void GetEnabledCloudApps(
+      std::vector<std::string>& enabled_apps) const = 0;
+
+  /**
+   * @brief Get cloud app policy information, all fields that aren't set for a
+   * given app will be filled with empty strings
+   * @param policy_app_id Unique application id
+   * @param endpoint Filled the endpoint used to connect to the cloud
+   * application
+   * @param certificate Filled with the certificate used to for creating a
+   * secure connection to the cloud application
+   * @param auth_token Filled with the token used for authentication when
+   * reconnecting to the cloud app
+   * @param cloud_transport_type Filled with the transport type used by the
+   * cloud application (ex. "WSS")
+   * @param cloud_transport_type Filled with the hybrid app preference for the
+   * cloud application set by the user
+   * @return true if the cloud app is enabled, false otherwise
```

Comment:
On second though, adding the parameter for all declarations and function calls would take a lot of time. If you can replace the function name with something more descriptive then that would be great otherwise it fine